.github/workflows/ci.yml

```diff
@@ -12,26 +12,51 @@ jobs:
   run-test:
     timeout-minutes: 15
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ["22.x"]
+    name: Build with Node ${{ matrix.node-version }}
     steps:
-      - name: Checkout Source
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
-      - name: Set up Node
-        uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
+      - name: Install pnpm
+        uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        # FIXME: does not work with pnpm
```

Comment:
@arunanshub Is this comment required? Is this something we need to revisit in future? If not, remove this